app/src/main/res/values/strings.xml

```diff
@@ -1763,4 +1763,7 @@
     <string name="pocket_stories_feature_caption">Part of the Firefox family. %s</string>
     <!-- Clickable text for opening an external link for more information about Pocket. -->
     <string name="pocket_stories_feature_learn_more">Learn more</string>
+    <string tools:ignore="UnusedResources" name="gdpr_has_you_covered_firefox_has_gdpr_covered">gdpr has you covered firefox has gdpr covered</string>
```

Comment:
Are these strings meant to be in this PR? They seem unrelated.

Reply:
They should be address on https://github.com/mozilla-mobile/android-components/pull/11879.
I was adding them to be able to compile locally